documents/Libraries/pbrlib/genglsl/mx_subsurfacebrdf.glsl

```diff
@@ -13,7 +13,21 @@ void mx_subsurfacebrdf_reflection(vec3 L, vec3 V, float weight, vec3 color, vec3
 }
 
 // Fake with simple diffuse transmission
-void mx_subsurfacebrdf_transmission(vec3 V, float weight, vec3 color, vec3 radius, float anisotropy, vec3 normal, out vec3 result)
+void mx_subsurfacebrdf_transmission(vec3 V, float weight, vec3 color, vec3 radius, float anisotropy, vec3 normal, BSDF bsdf, out vec3 result)
```

Comment:
The BSDF input should be removed here.

Reply:
Since the subsurface node is a pure reflection BSDF it doesn't need a transmission function, so the whole function can be removed here.
(It fakes subsurfacescattering by diffuse transmission but from the renderers perspective it's just a reflective BSDF)